src/qbittorrent.exe.manifest

```diff
@@ -1,5 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <assembly xmlns="urn:schemas-microsoft-com:asm.v1" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3" manifestVersion="1.0">
+  <assemblyIdentity
+    name="org.qbittorrent.qBittorrent"
```

Comment:
`version` doesn't matter in our case, right?

Reply:
yes.